### PR TITLE
Update /core page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Update /core page
- Add `/smartstart` and `tutorials` to the nav 
- Smart start bubble moved [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/9144)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/core
- Or view demo at: https://ubuntu-com-9142.demos.haus/core
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copydoc](https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit#heading=h.cc4g322ve6sn) 

## Issue / Card

Fixes [#3643](https://github.com/canonical-web-and-design/web-squad/issues/3643) and [#3668](https://github.com/canonical-web-and-design/web-squad/issues/3668)

### CODE REVIEW

**Comment by anasereijo** [comment]
🥇 LGTM

File: static/sass/_pattern_strip.scss [hunk 1]
```diff
@@ -625,7 +625,7 @@
         linear-gradient(251deg, #e95422 0%, #772953 49.5%, #2c001e 100%);
     }
 
-    background-position: top left, top right, center 67%, left top;
+    background-position: top left, top right, center 66.4%, left top;
```
Comment (pmahnke) [inline static/sass/_pattern_strip.scss:628]: I am seeing the black line now. 66.5 works better for me... test before you accept this.
```suggestion
    background-position: top left, top right, center 66.5%, left top;
```

File: templates/core/index.html [hunk 2]
```diff
@@ -6,127 +6,334 @@
...
-    <div class="col-6">
-      <h1>Build secure IoT devices  <br> with Ubuntu Core</h1>
-      <p class="p-heading--four u-sv3">Everything you love about Ubuntu, locked down for security. Helping you make safer things &ndash; because we're all connected.</p>
+    <div class="col-7">
+      <h1>Secure embedded devices</h1>
+      <p class="p-heading--four u-sv3">Give your device its own app store</p>
+      <p>
+        <a class="p-button--positive" href="/core/smartstart">Get to market faster</a>
+      </p>
+      <p>
```
Comment (pmahnke) [inline templates/core/index.html:17]:
```suggestion
      <p class="u-sv3">
```

I think there needs to be a bit more space,

**Review by pmahnke — CHANGES_REQUESTED** [review]
a few comments inline, very close

**Comment by SirSamTumless** [comment]
https://www.youtube.com/watch?v=qjmj2cy4LLA&feature=youtu.be Video here @bethcollins92

**Review by pmahnke — APPROVED** [review]
+1

**Pull request merged**